scripts/cnode-helper-scripts/guild-deploy.sh

```diff
@@ -229,7 +229,7 @@ os_dependencies() {
       pkg_list="${pkg_list} libusb ncurses-compat-libs pkgconfig"
     elif [[ "${VERSION_ID}" =~ "8" ]] || [[ "${VERSION_ID}" =~ "9" ]]; then
       #RHEL/CentOS/RockyLinux 8/9
-      pkg_opts="${pkg_opts} --allowerasing"
+      if ${pkgmgrcmd} -h  | grep -q "\--allowerasing"; then pkg_opts="${pkg_opts} --allowerasing"; fi
```

Comment:
This works for dnf v4 as the root usage menu displays this as a **General DNF option**. However, in dnf v5 `--allowerasing` no longer visible here, it is only visible in the usage for `dnf install`'s **Options**. 

This can be used to achieve your goal as both v4 and v5 list this in the usage menu for the install and update subcommands:

```suggestion
      if ${pkgmgrcmd} install -h  | grep -q "\--allowerasing"; then pkg_opts="${pkg_opts} --allowerasing"; fi
```